### ISSUE TITLE
Replace ender dragon model with our dragon models

### ISSUE THREAD

**Issue by AlexModGuy** [AlexModGuy/Ice-and-Fire-Suggestions#2]

**Comment by LordDecay26** [comment]
ummm.... OF COURSE

**Comment by WilliamBlaze** [comment]
i would opt this be taken one step further and give the ender dragon a proper boss fight mechanic to make it fight more like the other dragons from this mod; instead of its current "cheating ways" - as best said by notch in an old interview.

i mean, we've all fought it at one point or another and its kinda lame as is in vanilla.. so why not fix that by giving it an AI that similarly matches current stage five dragons? -- maybe boost its HP value and attack strength up to that of a maxed out tier 5 dragon by the player's default configuration settings.

It actually might be neat to raise an enderdragon from an egg; if the hatchling can be raised to grow up to be like the re-imagined version that this mod would make of it anyways.

also; just as to not break anything, if we are infact given the ability to loot the enderdragon after killing it like any other dragon, it should probly be coded in that the normal event that is trigger whenever the enderdragon dies in vannilla be auto trigger "after" its corpse is fully looted.

again, that's if were going with the idea that the player is going to get an enderdragon they can raise from killing the enderdragon with this mod installed. but now i'm just getting ahead of things with my crazy train now aren't i?

**Comment by LordDecay26** [comment]
i like all the ideas except for the one where making the dragon have a corpse like all the other modded dragons, to me, that would remove the mysticism that comes with the dragon, i like how currently it "returns to sender" as in, dissipates awesomely through a portal of mystic proportions. then dropping all the experience and an egg, magically appears atop the dragons nest of bedrock, which also seems to be your route home. i like the idea of that magical egg being able to be hatched, too. 't'would be neat if you could only hatch it if it is on a bedrock block (so basically you have to hatch it at the bottom of the world in order for it to hatch) MAKE IT HAPPEN ALEX

**Comment by kuroyure** [comment]
cool idea ,it would be cool too if they added a new model to the dragon in the i&f style

**Comment by Rutes** [comment]
I'm all for the replacement of Ender dragons with I&F dragon model - maybe it could be a new dragon type!

**Comment by exmortis314** [comment]
DEWIT

**Comment by seymourimadeit** [comment]
ender dragons is a alien from another dimesion or planet, of course its gonna look different from earth dragons.


maybe just add it in as a config file and have it defaultlly set to false

**Comment by nikolas2571** [comment]
If the ender dragon were to be fully made into an Ice and Fire dragon and thus follow Ice and Fire dragon colouration rules, I would like to suggest "Ender Black", "Obsidian Black", "Iron Black" (a reference to the process of "blueing"/"blacking" iron or steel), and "Void Black"!

**Comment by KoradTheSinner** [comment]
absolutely. all the people who use this mod want the ender dragon to be different.

**Comment by chcknlvr** [comment]
Great idea I think it would be awesome to have ice and fire AI.

**Comment by M0ltenM3tal** [comment]
WilliamBlaze, I absolutely love your idea of the Ender Dragon change concept, I was even thinking about it myself, but obviously not as hard or as well planned as you did. I do have some suggestions as for fighting the Ender Dragon and some other things about 'her' and 'her' kind:

1. The 1st Ender Dragon that starts in the center should be a Empress Dragon that is Jet Black, because it is stated in the minecraft wiki that the Ender dragon is a female, is originally black in color, and being the end-game dragon, it should be a Special Ender Dragon (a tier above tier 5) to make things more challenging, and you wouldn't have to change the scaling of health and strength of the basic Ender Dragons. 
2. If the death would be leaving a skeleton, maybe make the skeleton of the ender dragons a unique color to match the theme of the end.
3. There should be other Ender Dragon nests scattered across the end ring, consisting of level 1-3 dragons, and having level 4-5 dragons underground, or if the keep would be too small for the dragon, have a custom void-like keep above or below the end island ring.
4. The new way to summon this new Ender Dragon Empress' successor could be changed to use the skull of a tier-5 dragon, Ender Dragon Bone Blocks, Ender Dragon Block of Scales (of any color), and the original, End Crystals. This would make re-summoning the End Boss more interesting.
5. This new re-summoned Ender Dragon would be a Ender Dragon Prince/Princess. The Ender Dragon Prince/Princess could have alternative scale colors, and would be easier than the Empress, but less rewarding than the Empress (just like killing the vanilla ender dragon again, except the fight isn't the same for less reward). They would still be harder than the tier 5 Ender Dragons.

These are just additional ideas that I had for this topic. The traditional way for her death would cause some issues in the way to collect resources. how would collecting Ender Dragon Blood work? How would dropping resources work? Would you get only some flesh, some bone, and some scales? That's why I like the idea of leaving a skeleton behind, then activate the action that would have played when it is looted.

**Comment by gaesyeah** [comment]
https://www.curseforge.com/minecraft/texture-packs/better-ender-dragon 
Well, we already have it, its just a resource pack, but looks really good.

**Comment by M0ltenM3tal** [comment]
Just from looking at the pictures alone, I can see why people would think this looks amazing, I mean, the details of the black scaled dragon to have fade-in purple spikes/horns/claws is really nice. The work and love put into this must be a lot. Even remodeling the dragon’s head is a nice touch. :D

**Comment by SlushieGamez** [comment]
YES PLZ

**Comment by Aceplante** [comment]
Why is this 6 years old and still not added yet... this is such an obvious and perfect addition, it shouldve been done a long time ago.. Also, I do want a body on the ground, becuase then it would perfectly integrate with ends delight

**Comment by ali-afk** [comment]
Alex, when is this coming into I&F 🫠
